components/Login/index.js

```diff
@@ -0,0 +1,33 @@
+import { signIn, signOut, useSession } from "next-auth/react";
+import {
+  LoginContainer,
+  LoginButton,
+  LoginText,
+  LogoutButton,
+  AdminButton,
+} from "./styles";
+import Router from "next/router";
+
+export default function Login() {
+  const router = Router;
+  const { data: session } = useSession();
+
+  if (session) {
+    return (
+      <LoginContainer>
+        <LoginText>Eingeloggt als {session.user.name}</LoginText>
+        <LogoutButton onClick={signOut}>Log Out</LogoutButton>
+        <AdminButton onClick={() => router.push("/admin")}>
```

Comment:
since you are Linking to another page in your app, This should be a Link.

Reply:
Thx, changed it to this: `<Link href="/admin">
          <AdminButton>Go to Admin View</AdminButton>
        </Link>`

Reply:
Why do you like that 'button in a link' so much? 😀

Reply:
I think I just liked the fact that it worked. But now I finally have to let it go. Changed it.